osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -821,15 +821,13 @@ public static async Task<BeatmapSetInfo> LoadOszIntoOsu(OsuGameBase osu, string
 
             var manager = osu.Dependencies.Get<BeatmapManager>();
 
-            await manager.Import(temp);
-
-            var imported = manager.GetAllUsableBeatmapSets();
+            var importedSet = await manager.Import(temp);
 
             ensureLoaded(osu);
 
             waitForOrAssert(() => !File.Exists(temp), "Temporary file still exists after standard import", 5000);
 
-            return imported.LastOrDefault();
+            return manager.GetAllUsableBeatmapSets().Find(beatmapSet => beatmapSet.ID == importedSet.ID);
```

Comment:
could probably use `manager.QueryBeatmapSets(s => s.ID == importedSet.ID).First()` to avoid pulling every beatmap set each time into a list, but not sure it matters much in a test context.

Reply:
Fair point, should be equivalent as far as I can tell, but yeah, probably can be done if it's ever a perf problem.

Reply:
usable checks for deletion, which query doesn't. kinda weird api we have there.